### DISCUSSION TITLE
Can I download a track using the track's "uri" property or any other way?

### DISCUSSION THREAD

**Discussion by TechPro424** [backtrackapp/spotify.js#40] [Q&A] [answered]
I want to download a track because to use it for my music bot, and the package I currently use scrapes from YT, so if I were to monetize my bot, it would be shut down by YT. Can I download a track using the track's "uri" property or any other way?

**Comment by Martijn-Faber** [comment, marked as answer]
No. Unfortunately not

**Reply by TechPro424** [reply]
Sad to hear, but understandable. Thanks for the answer!

**Comment by stingalleman** [comment]
Yeah, you can, but it would be illegal. Google is your friend :)
